worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/ActionTypeAdapter.java

```diff
@@ -0,0 +1,40 @@
+package com.github.cafdataprocessing.workflow;
+
+import com.github.cafdataprocessing.workflow.model.Action;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+import java.io.IOException;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+
+public class ActionTypeAdapter extends TypeAdapter<Action>
+{
+    private final Gson gson= new GsonBuilder().setPrettyPrinting().create();
+    @Override
+    public void write(JsonWriter out, Action action) throws IOException
+    {
+        final String queueName = StringUtils.isEmpty(System.getenv("CAF_WORKFLOW_ACTION_"
+                    + action.getName().toUpperCase() + "_INPUT_QUEUE"))
```

Comment:
You are not replacing the _ in the action name here. 
 StringUtils.isEmpty(System.getenv("CAF_WORKFLOW_ACTION_"
                    + action.getName().toUpperCase() + "_INPUT_QUEUE"))

We need to test this behaviour.